### ISSUE TITLE
[Bug] Text is aligned to the right on the latest about page

### ISSUE THREAD

**Issue by xavier630** [kblincoe/QualOpt_SE701#140]
On the latest about page, the text is all towards the right side of the screen, at least on medium viewports.
Here is a screenshot:
![image](https://user-images.githubusercontent.com/9077250/37930857-76cf199e-31a0-11e8-940a-9d6f23cef071.png)

It is very unconventional in web development to use only the right two-thirds of the page when placing text.

I estimate that this would be a very small issue to fix.

**Comment by softeng-701** [comment]
@xavier630 Do you think the work takes 1 hour? Please comment! Thanks

**Comment by xavier630** [comment]
@softeng-701 
Yep, I imagine it would take about as a rough estimate. If you weren't familiar with the page and needed to get up to speed, it would potentially take a bit longer. One could possibly argue as well that it's a small rather than an extra small if that's what you're getting at.

Cheers.

**Comment by softeng-701** [comment]
@xavier630 go for it! APPROVED!
